<a href="https://colab.research.google.com/github/Sedowlas/EDA/blob/main/House_Rent(EDA%2CPD%2CSQL).ipynb" target="_parent"><img src="https://colab.research.google.com/assets/colab-badge.svg" alt="Open In Colab"/></a>

#Легенда

BHK: количество спален, прихожей, кухни.  
Аренда: Аренда домов / квартир / квартир.  
Размер: размер домов / квартир / квартир в квадратных футах.  
Этаж: дома / Квартиры / Квартиры, расположенные на каком этаже, и общее количество этажей (пример: первый этаж из 2, 3 из 5 и т. Д.)  
Тип района: размер домов / квартир / квартир, рассчитанный либо по Супер площади, либо по площади ковра, либо по площади застройки.  
Местоположение района: местоположение домов / квартир / квартир.  
Город: город, в котором расположены дома / квартиры / квартиры.  
Статус меблировки: статус меблировки домов / квартир / квартир, либо они меблированы, либо частично меблированы, либо без мебели.  
Предпочитаемый арендатор: тип арендатора, предпочитаемый владельцем или агентом.  
Ванная комната: количество ванных комнат.  
Контактное лицо: С кем вам следует связаться для получения дополнительной информации о Домах / Квартирах / Квартирах.  

#Загрузка данных

https://drive.google.com/file/d/1cihcd4PuQS11y-IV1YDcmBHgT5BiIMyW/view?usp=sharing

In [43]:
!gdown 1cihcd4PuQS11y-IV1YDcmBHgT5BiIMyW

Downloading...
From: https://drive.google.com/uc?id=1cihcd4PuQS11y-IV1YDcmBHgT5BiIMyW
To: /content/House_Rent_Dataset.csv
100% 567k/567k [00:00<00:00, 90.2MB/s]


#Предобработка

In [44]:
import pandas as pd
import numpy as np

In [45]:
import sqlite3

In [46]:
df = pd.read_csv('/content/House_Rent_Dataset.csv')

In [47]:
con = sqlite3.connect('db')

In [48]:
df.head()

Posted On  BHK   Rent  Size            Floor    Area Type  \
0  2022-05-18    2  10000  1100  Ground out of 2   Super Area   
1  2022-05-13    2  20000   800       1 out of 3   Super Area   
2  2022-05-16    2  17000  1000       1 out of 3   Super Area   
3  2022-07-04    2  10000   800       1 out of 2   Super Area   
4  2022-05-09    2   7500   850       1 out of 2  Carpet Area   

              Area Locality     City Furnishing Status  Tenant Preferred  \
0                    Bandel  Kolkata       Unfurnished  Bachelors/Family   
1  Phool Bagan, Kankurgachi  Kolkata    Semi-Furnished  Bachelors/Family   
2   Salt Lake City Sector 2  Kolkata    Semi-Furnished  Bachelors/Family   
3               Dumdum Park  Kolkata       Unfurnished  Bachelors/Family   
4             South Dum Dum  Kolkata       Unfurnished         Bachelors   

   Bathroom Point of Contact  
0         2    Contact Owner  
1         1    Contact Owner  
2         1    Contact Owner  
3         1    Contact Owner  
4         1    Contact Owner

In [49]:
len(df)

4746

In [50]:
df.columns = df.columns.str.lower().str.replace(' ','_')

In [51]:
df.isna().mean()

posted_on            0.0
bhk                  0.0
rent                 0.0
size                 0.0
floor                0.0
area_type            0.0
area_locality        0.0
city                 0.0
furnishing_status    0.0
tenant_preferred     0.0
bathroom             0.0
point_of_contact     0.0
dtype: float64

In [52]:
df.dtypes

posted_on            object
bhk                   int64
rent                  int64
size                  int64
floor                object
area_type            object
area_locality        object
city                 object
furnishing_status    object
tenant_preferred     object
bathroom              int64
point_of_contact     object
dtype: object

In [53]:
df.to_sql('house',con,index=False,if_exists='replace')

In [54]:
def select(sql):
  return pd.read_sql(sql,con)

In [85]:
pd.set_option('display.max_rows', 20)

#Анализ

In [55]:
df.head()

posted_on  bhk   rent  size            floor    area_type  \
0  2022-05-18    2  10000  1100  Ground out of 2   Super Area   
1  2022-05-13    2  20000   800       1 out of 3   Super Area   
2  2022-05-16    2  17000  1000       1 out of 3   Super Area   
3  2022-07-04    2  10000   800       1 out of 2   Super Area   
4  2022-05-09    2   7500   850       1 out of 2  Carpet Area   

              area_locality     city furnishing_status  tenant_preferred  \
0                    Bandel  Kolkata       Unfurnished  Bachelors/Family   
1  Phool Bagan, Kankurgachi  Kolkata    Semi-Furnished  Bachelors/Family   
2   Salt Lake City Sector 2  Kolkata    Semi-Furnished  Bachelors/Family   
3               Dumdum Park  Kolkata       Unfurnished  Bachelors/Family   
4             South Dum Dum  Kolkata       Unfurnished         Bachelors   

   bathroom point_of_contact  
0         2    Contact Owner  
1         1    Contact Owner  
2         1    Contact Owner  
3         1    Contact Owner  
4         1    Contact Owner

In [56]:
df.groupby(['city'])[['rent']].agg(['count','mean'])

rent              
          count          mean
city                         
Bangalore   886  24966.365688
Chennai     891  21614.092031
Delhi       605  29461.983471
Hyderabad   868  20555.048387
Kolkata     524  11645.173664
Mumbai      972  85321.204733

Самые дорогие квартиры, дома находятся в городе Mumbai

In [57]:
df.groupby(['city'])[['size']].agg(['count','mean'])

size             
          count         mean
city                        
Bangalore   886   985.925508
Chennai     891  1031.713805
Delhi       605   786.406612
Hyderabad   868  1186.669355
Kolkata     524   787.366412
Mumbai      972   905.904321

В среднем, самые большие квартиры в квадратных метрах находятся в городе Hyderabad. Но судя по предыдущей таблице, мы видим, что цены на квартиры, которые находятся в городе Hyderabad намного ниже, чем все остальные города. Следовательно делаем вывод, что в городе Hyderabad квартиры плохого качества, мало комнат либо же квартиры находятся в неблагополучных районах!

In [58]:
df.groupby(['city'])[['bhk']].agg(['count','mean'])

bhk          
          count      mean
city                     
Bangalore   886  1.930023
Chennai     891  2.124579
Delhi       605  2.114050
Hyderabad   868  2.206221
Kolkata     524  1.965649
Mumbai      972  2.122428

Видим что в квартирах города Hyderabad, в среднем комнат больше, чем во всех остальных городах! То есть вариант с малым количеством комнат мы отсеиваем.  
В Mumbai количество комнат, в сравнении с отсальными городами, среднее.

In [59]:
df.groupby(['city'])[['bathroom']].agg(['count','mean'])

bathroom          
             count      mean
city                        
Bangalore      886  1.793454
Chennai        891  1.998878
Delhi          605  1.852893
Hyderabad      868  2.149770
Kolkata        524  1.431298
Mumbai         972  2.287037

Наблюдаем, что в городе Hyderabad, в среднем большинство квартир с 2 ванными комнатами.  
В городе Mumbai большая часть квартир с 2 и более комнатами!

In [61]:
df.groupby(['city','area_locality'])[['rent']].agg(['count','mean'])

rent              
                                                        count          mean
city      area_locality                                                    
Bangalore A Narayanapura, Mahadevapura                      2  14500.000000
          Aarna Enclave                                     1  90000.000000
          Abbiareddy Layout, Kaggadasapura                  1  22500.000000
          Abbigere                                          3   9333.333333
          Aditya Nagar-Vidyaranyapura, Vidyaranyapura       1  10000.000000
...                                                       ...           ...
Mumbai    simla house at nepeasea road, Napean Sea Road     1  45000.000000
          spring grove uno, Lokhandwala Twp                 1  18000.000000
          sra                                               1  15000.000000
          sunanda circal                                    1   7500.000000
          vanamali chs ghatla, Ghatla                       1  52000.000000

[2240 rows x 2 columns]

Что бы узнать какая самая дорогая квартира, в представленных городах воспользуемся SQL, для быстрого вычесления

In [62]:
sql = '''select t.city, t.area_locality, t.rent from house t
order by t.rent desc
'''

In [63]:
select(sql)

city            area_locality     rent
0     Bangalore             Marathahalli  3500000
1        Mumbai                     Juhu  1200000
2        Mumbai    Raheja Artesia, Worli  1000000
3        Mumbai             Breach Candy   850000
4        Mumbai  Lady Ratan Tower, Worli   700000
...         ...                      ...      ...
4741    Kolkata                Baranagar     2200
4742      Delhi                Ram Nagar     2000
4743    Kolkata              Shyam Bazar     1800
4744    Kolkata               Santoshpur     1500
4745  Hyderabad            Uppal, NH 2 2     1200

[4746 rows x 3 columns]

Узнали что самая дорогая квартира в городе Mumbai, в районе Juhu стоит 1200000, а в городе Hyderabad, в районе Jubilee Hills 400000.

In [64]:
df.groupby(['city','area_locality'])[['size']].agg(['count','mean'])

size        
                                                        count    mean
city      area_locality                                              
Bangalore A Narayanapura, Mahadevapura                      2   900.0
          Aarna Enclave                                     1   950.0
          Abbiareddy Layout, Kaggadasapura                  1  1115.0
          Abbigere                                          3   700.0
          Aditya Nagar-Vidyaranyapura, Vidyaranyapura       1   300.0
...                                                       ...     ...
Mumbai    simla house at nepeasea road, Napean Sea Road     1   225.0
          spring grove uno, Lokhandwala Twp                 1   320.0
          sra                                               1   380.0
          sunanda circal                                    1   380.0
          vanamali chs ghatla, Ghatla                       1   750.0

[2240 rows x 2 columns]

In [65]:
sql = '''select t.city, t.area_locality, t.size from house t
order by t.size desc
'''

In [66]:
select(sql)

city                         area_locality  size
0     Hyderabad   Beeramguda, Ramachandra Puram, NH 9  8000
1     Hyderabad                         Jubilee Hills  7000
2       Chennai                               Navalur  6000
3     Bangalore                            whitefield  5700
4        Mumbai                                  Juhu  5000
...         ...                                   ...   ...
4741      Delhi                                Burari    25
4742      Delhi                     Hari Nagar Ashram    25
4743  Hyderabad                    Sanath Nagar, NH 9    25
4744    Kolkata                          Dakshineswar    20
4745  Hyderabad                     Ghatkesar, NH 2 2    10

[4746 rows x 3 columns]

Узнали что самая большая квартира из всех представленных находится в городе Hyderabad, в районе Beeramguda, Ramachandra Puram, NH 9 которая составляет 8000 кв.фунтов.  
Самая большая квартира в городе Mumbai, в районе Juhu составляет 5000 кв.фунтов

In [67]:
df.groupby(['city','area_locality'])[['bhk']].agg(['count','mean'])

bhk          
                                                        count      mean
city      area_locality                                                
Bangalore A Narayanapura, Mahadevapura                      2  1.500000
          Aarna Enclave                                     1  2.000000
          Abbiareddy Layout, Kaggadasapura                  1  2.000000
          Abbigere                                          3  1.666667
          Aditya Nagar-Vidyaranyapura, Vidyaranyapura       1  2.000000
...                                                       ...       ...
Mumbai    simla house at nepeasea road, Napean Sea Road     1  1.000000
          spring grove uno, Lokhandwala Twp                 1  1.000000
          sra                                               1  1.000000
          sunanda circal                                    1  1.000000
          vanamali chs ghatla, Ghatla                       1  2.000000

[2240 rows x 2 columns]

In [68]:
sql = '''select t.city, t.area_locality, t.bhk from house t
order by t.bhk desc
'''

In [69]:
select(sql)

city                     area_locality  bhk
0       Kolkata                          Sonarpur    6
1       Chennai              Raja Annamalai Puram    6
2       Chennai                      Besant Nagar    6
3     Hyderabad                  Boduppal, NH 2 2    6
4     Hyderabad  Mallikarjuna Nagar, Secunderabad    6
...         ...                               ...  ...
4741  Hyderabad                           Nagaram    1
4742  Hyderabad                   Jagadgiri Gutta    1
4743  Hyderabad                         Hyderguda    1
4744  Hyderabad                Basheer Bagh, NH 7    1
4745  Hyderabad                    BN Reddy Nagar    1

[4746 rows x 3 columns]

Макисмальное количество комнат составляет 6.  
В городе Hyderabad есть 5 квартир с 6 комнатами в районах: Boduppal, NH 2 2 / Mallikarjuna Nagar, Secunderabad / Rampally / Kakateeya Nagar, Secunderabad	/ Kuntloor.  
В городе Mumbai максимальное количество комнат 5, есть 10 квартир с 5 комнатами в районах: Runwal Elegante, Andheri West / Runwal Elegante, Andheri West / Bandra East / Bandra West / Chandivali / Altamount Road / Deonar / Runwal Elegante, Andheri West / Khar West	/ Bandra West / MidCity Shikhar, Andheri West.

In [70]:
df.groupby(['city','area_locality'])[['bathroom']].agg(['count','mean'])

bathroom          
                                                           count      mean
city      area_locality                                                   
Bangalore A Narayanapura, Mahadevapura                         2  1.500000
          Aarna Enclave                                        1  2.000000
          Abbiareddy Layout, Kaggadasapura                     1  2.000000
          Abbigere                                             3  1.666667
          Aditya Nagar-Vidyaranyapura, Vidyaranyapura          1  2.000000
...                                                          ...       ...
Mumbai    simla house at nepeasea road, Napean Sea Road        1  1.000000
          spring grove uno, Lokhandwala Twp                    1  1.000000
          sra                                                  1  1.000000
          sunanda circal                                       1  2.000000
          vanamali chs ghatla, Ghatla                          1  2.000000

[2240 rows x 2 columns]

In [71]:
sql = '''select t.city, t.area_locality, t.bathroom from house t
order by t.bathroom desc
'''

In [72]:
select(sql)

city                         area_locality  bathroom
0     Hyderabad   Beeramguda, Ramachandra Puram, NH 9        10
1         Delhi                              West End         7
2     Hyderabad                      Boduppal, NH 2 2         7
3     Hyderabad                              Rampally         7
4        Mumbai        Island City Centre, Dadar East         6
...         ...                                   ...       ...
4741  Hyderabad                             Hyderguda         1
4742  Hyderabad          Nallagandla, Serilingampally         1
4743  Hyderabad                    Basheer Bagh, NH 7         1
4744  Hyderabad                        Vinayaka Nagar         1
4745  Hyderabad                        BN Reddy Nagar         1

[4746 rows x 3 columns]

Максимальное количество ванных комнат составляет 10.  
В городе Hyderabad, в районе Beeramguda, Ramachandra Puram, NH 9 есть одна квартира с 10 ванными комнатами.  
В городе Mumbai максимальное количество ванных комнат 6, есть 4 квартир с 6 ванными комнатами в районах: Island City Centre, Dadar East	/ Windsor Grande Residences, Andheri West	/ Oberoi Prisma, Agarkar Chowk / Bombay Dyeing Two ICC, Dadar TT Circle.

In [74]:
df.groupby(['city','area_type'])[['rent']].agg(['count','mean'])

rent              
                      count          mean
city      area_type                      
Bangalore Carpet Area   336  39998.214286
          Super Area    550  15783.272727
Chennai   Built Area      1  15000.000000
          Carpet Area   315  28899.342857
          Super Area    575  17634.544348
Delhi     Carpet Area   315  33907.298413
          Super Area    290  24633.451724
Hyderabad Built Area      1   6000.000000
          Carpet Area   246  28739.024390
          Super Area    621  17336.524960
Kolkata   Carpet Area   264  12263.636364
          Super Area    260  11017.196154
Mumbai    Carpet Area   822  93493.795620
          Super Area    150  40535.406667

Узнали что в городах Hyderabad и Mumbai в среднем дорогие квартиры зависят от площади ковра.

In [75]:
df.groupby(['city','area_type'])[['size']].agg(['count','mean'])

size             
                      count         mean
city      area_type                     
Bangalore Carpet Area   336  1132.791667
          Super Area    550   896.203636
Chennai   Built Area      1  1000.000000
          Carpet Area   315  1138.688889
          Super Area    575   973.165217
Delhi     Carpet Area   315   917.765079
          Super Area    290   643.724138
Hyderabad Built Area      1   500.000000
          Carpet Area   246  1397.028455
          Super Area    621  1104.444444
Kolkata   Carpet Area   264   739.007576
          Super Area    260   836.469231
Mumbai    Carpet Area   822   945.357664
          Super Area    150   689.700000

Узнали что в городах Hyderabad и Mumbai в среднем большие квартиры в кв.фунтах зависят от площади ковра.

In [76]:
df.groupby(['city','area_type'])[['bhk']].agg(['count','mean'])

bhk          
                      count      mean
city      area_type                  
Bangalore Carpet Area   336  2.086310
          Super Area    550  1.834545
Chennai   Built Area      1  2.000000
          Carpet Area   315  2.333333
          Super Area    575  2.010435
Delhi     Carpet Area   315  2.222222
          Super Area    290  1.996552
Hyderabad Built Area      1  1.000000
          Carpet Area   246  2.447154
          Super Area    621  2.112721
Kolkata   Carpet Area   264  1.950758
          Super Area    260  1.980769
Mumbai    Carpet Area   822  2.239659
          Super Area    150  1.480000

Узнали что в городах Hyderabad и Mumbai в среднем квартиры у которых больше комнат зависят от площади ковра.

In [77]:
df.groupby(['city','area_type'])[['bathroom']].agg(['count','mean'])

bathroom          
                         count      mean
city      area_type                     
Bangalore Carpet Area      336  1.949405
          Super Area       550  1.698182
Chennai   Built Area         1  2.000000
          Carpet Area      315  2.193651
          Super Area       575  1.892174
Delhi     Carpet Area      315  2.019048
          Super Area       290  1.672414
Hyderabad Built Area         1  1.000000
          Carpet Area      246  2.369919
          Super Area       621  2.064412
Kolkata   Carpet Area      264  1.424242
          Super Area       260  1.438462
Mumbai    Carpet Area      822  2.388078
          Super Area       150  1.733333

Узнали что в городах Hyderabad и Mumbai в среднем квартиры у которых большее количество ванных комнат зависят от площади ковра.

In [79]:
df.groupby(['city','furnishing_status'])[['rent']].agg(['count','mean'])

rent               
                            count           mean
city      furnishing_status                     
Bangalore Furnished            91   32531.868132
          Semi-Furnished      584   26879.452055
          Unfurnished         211   16408.530806
Chennai   Furnished            81   32961.666667
          Semi-Furnished      448   23900.582589
          Unfurnished         362   16245.303867
Delhi     Furnished            97   30214.443299
          Semi-Furnished      278   33945.320144
          Unfurnished         230   23725.652174
Hyderabad Furnished           111   24353.153153
          Semi-Furnished      402   25002.442786
          Unfurnished         355   14331.267606
Kolkata   Furnished            61   13865.573770
          Semi-Furnished      142   11722.535211
          Unfurnished         321   11189.006231
Mumbai    Furnished           239  108974.527197
          Semi-Furnished      397   99744.586902
          Unfurnished         336   51454.458333

Узнали что в городе Hyderabad, больше всего квартир Полу-Меблированных в количестве 402 квартиры и в среднем они стоят больше, чем Обставленные, но не на много.  
В городе Mumbai, так же больше всего квартир Полу-Мебелированные в количестве 397 квартир, но стоят они дешевле чем Обставленные!

In [80]:
df.groupby(['city','furnishing_status'])[['size']].agg(['count','mean'])

size             
                            count         mean
city      furnishing_status                   
Bangalore Furnished            91  1199.659341
          Semi-Furnished      584  1013.522260
          Unfurnished         211   817.364929
Chennai   Furnished            81  1233.839506
          Semi-Furnished      448  1128.466518
          Unfurnished         362   866.748619
Delhi     Furnished            97   690.329897
          Semi-Furnished      278   936.395683
          Unfurnished         230   645.634783
Hyderabad Furnished           111  1296.099099
          Semi-Furnished      402  1385.634328
          Unfurnished         355   927.146479
Kolkata   Furnished            61   793.213115
          Semi-Furnished      142   824.929577
          Unfurnished         321   769.638629
Mumbai    Furnished           239  1020.493724
          Semi-Furnished      397   990.342569
          Unfurnished         336   724.627976

Узнали что в городе Hyderabad, Полу-Меблированные квартиры в среднем больше в кв.фунтах, чем Обставленные.  
В городе Mumbai, Обставленные квартиры больше в кв.фунтах, чем Полу-Мебелированные!

In [81]:
df.groupby(['city','furnishing_status'])[['bhk']].agg(['count','mean'])

bhk          
                            count      mean
city      furnishing_status                
Bangalore Furnished            91  2.197802
          Semi-Furnished      584  1.969178
          Unfurnished         211  1.706161
Chennai   Furnished            81  2.320988
          Semi-Furnished      448  2.274554
          Unfurnished         362  1.895028
Delhi     Furnished            97  1.814433
          Semi-Furnished      278  2.262590
          Unfurnished         230  2.060870
Hyderabad Furnished           111  2.315315
          Semi-Furnished      402  2.422886
          Unfurnished         355  1.926761
Kolkata   Furnished            61  1.819672
          Semi-Furnished      142  1.992958
          Unfurnished         321  1.981308
Mumbai    Furnished           239  2.271967
          Semi-Furnished      397  2.264484
          Unfurnished         336  1.848214

Узнали что в городе Hyderabad, у Полу-Меблированных квартиры в среднем больше комнат, чем в Обставленных квартирах.  
В городе Mumbai, у Обставленных квартир в среднем комнат больше, чем Полу-Мебелированных!

In [82]:
df.groupby(['city','furnishing_status'])[['bathroom']].agg(['count','mean'])

bathroom          
                               count      mean
city      furnishing_status                   
Bangalore Furnished               91  2.076923
          Semi-Furnished         584  1.820205
          Unfurnished            211  1.597156
Chennai   Furnished               81  2.209877
          Semi-Furnished         448  2.145089
          Unfurnished            362  1.770718
Delhi     Furnished               97  1.649485
          Semi-Furnished         278  2.043165
          Unfurnished            230  1.708696
Hyderabad Furnished              111  2.270270
          Semi-Furnished         402  2.338308
          Unfurnished            355  1.898592
Kolkata   Furnished               61  1.475410
          Semi-Furnished         142  1.500000
          Unfurnished            321  1.392523
Mumbai    Furnished              239  2.414226
          Semi-Furnished         397  2.471033
          Unfurnished            336  1.979167

Узнали что в городе Hyderabad, у Полу-Меблированных квартиры в среднем ванных комнат больше, чем в Обставленных квартирах.  
В городе Mumbai, у Обставленных квартир в среднем ванных комнат меньше, чем у Полу-Мебелированных!

In [86]:
(df.groupby(['city','area_locality','area_type','furnishing_status','floor','tenant_preferred','point_of_contact'])
[['rent','size','bhk','bathroom']]
 .agg({'rent':['count','mean'],
       'size':['count','mean'],
       'bhk':['count','mean'],
       'bathroom':['count','mean']}))

rent  \
                                                                                                                                        count   
city      area_locality                                 area_type   furnishing_status floor           tenant_preferred point_of_contact         
Bangalore A Narayanapura, Mahadevapura                  Carpet Area Semi-Furnished    5 out of 5      Bachelors/Family Contact Agent        1   
                                                        Super Area  Semi-Furnished    Ground out of 3 Bachelors/Family Contact Owner        1   
          Aarna Enclave                                 Carpet Area Unfurnished       4 out of 5      Bachelors/Family Contact Owner        1   
          Abbiareddy Layout, Kaggadasapura              Super Area  Semi-Furnished    1 out of 4      Bachelors/Family Contact Owner        1   
          Abbigere                                      Carpet Area Semi-Furnished    1 out of 3      Bachelors/Family Contact Owner        1   
...                                                                                                                                       ...   
Mumbai    simla house at nepeasea road, Napean Sea Road Carpet Area Unfurnished       3 out of 6      Bachelors        Contact Owner        1   
          spring grove uno, Lokhandwala Twp             Carpet Area Furnished         5 out of 7      Bachelors/Family Contact Agent        1   
          sra                                           Carpet Area Unfurnished       19 out of 19    Bachelors/Family Contact Owner        1   
          sunanda circal                                Carpet Area Unfurnished       3 out of 7      Bachelors        Contact Owner        1   
          vanamali chs ghatla, Ghatla                   Carpet Area Unfurnished       4 out of 20     Bachelors/Family Contact Agent        1   

                                                                                                                                                  \
                                                                                                                                            mean   
city      area_locality                                 area_type   furnishing_status floor           tenant_preferred point_of_contact            
Bangalore A Narayanapura, Mahadevapura                  Carpet Area Semi-Furnished    5 out of 5      Bachelors/Family Contact Agent     15000.0   
                                                        Super Area  Semi-Furnished    Ground out of 3 Bachelors/Family Contact Owner     14000.0   
          Aarna Enclave                                 Carpet Area Unfurnished       4 out of 5      Bachelors/Family Contact Owner     90000.0   
          Abbiareddy Layout, Kaggadasapura              Super Area  Semi-Furnished    1 out of 4      Bachelors/Family Contact Owner     22500.0   
          Abbigere                                      Carpet Area Semi-Furnished    1 out of 3      Bachelors/Family Contact Owner     12000.0   
...                                                                                                                                          ...   
Mumbai    simla house at nepeasea road, Napean Sea Road Carpet Area Unfurnished       3 out of 6      Bachelors        Contact Owner     45000.0   
          spring grove uno, Lokhandwala Twp             Carpet Area Furnished         5 out of 7      Bachelors/Family Contact Agent     18000.0   
          sra                                           Carpet Area Unfurnished       19 out of 19    Bachelors/Family Contact Owner     15000.0   
          sunanda circal                                Carpet Area Unfurnished       3 out of 7      Bachelors        Contact Owner      7500.0   
          vanamali chs ghatla, Ghatla                   Carpet Area Unfurnished       4 out of 20     Bachelors/Family Contact Agent     52000.0   

                                 